tutorials/kubernetes/install-kubernetes-with-conjure-up.md

```diff
@@ -0,0 +1,159 @@
+---
+id: install-kubernetes-with-conjure-up
+summary: This tutorial will guide you through the installation of the Canonical Distribution of Kubernetes with a few simple commands using conjure-up.
+categories: kubernetes
+status: Published
+feedbacklink: https://github.com/ubuntu/codelabs/issues
+tags: kubernetes, conjure-up
+difficulty: 3
+published: 2017-08-21
+
```

Comment:
Please add an `author` field.
If you don't wish to be the designated maintainer in the long run, you can use `
author: Canonical Web Team <webteam@canonical.com>`

Reply:
Added